### ISSUE TITLE
[Q] How can I create multiple logs starting at step 0 one after the other?

### ISSUE THREAD

**Issue by Janik7777** [wandb/wandb#5819]
I am currently working on implementing the Forward Forward Algorithm. The idea is that you don't train the whole net by backpropagation, but each layer at a time and unsupervised. I now want to log the loss of each layer one after the other. This works fine using
```python
      for i in tqdm(range(self.num_epochs)):
            #train...
            wandb.log({"loss_" + self.name: loss.item()})
```
But now the logs for "loss_0" will be for steps 0..num_epochs-1, the logs for "loss_0" will be for steps num_epochs..2num_epochs-1 and so on.

This now gets problematic if I want to compare runs with different num_epochs, because for all but "loss_0" the start point is a different one and the plots are messed up:
<img width="409" alt="Screenshot 2023-06-30 at 12 20 02" src="https://github.com/wandb/wandb/assets/92917297/44ee0bc4-7489-4dbf-9832-d1a81c8fe6e9">

Is there a way to start at step 0 again for the new plot or can I display the plots such that they start at the same point?

I also found #3890 stating I can't get back to step 0...

**Comment by rsanandres-wandb** [comment]
I would recommend logging with a custom step when trying to implement Forward Forward Algorithm. For the following example, `loss_0` logs to the `_step` but `loss_1` logs to `custom_step`, which can be set.

```
wandb.define_metric("custom_step")
for i in range(5):
    wandb.log({"loss_0" : loss_0})
    wandb.log({"loss_1" : loss_1, "custom_step" : i)
```

However, since the metrics were logged to the universal value `_step`, there isn't a clear way to adjust the x-axis since they all are using `step`.

**Comment by Janik7777** [comment]
Thank you. Using this I can just log the custom step and after setting it as x_value in the UI, it is displayed perfectly.
```python
      for i in tqdm(range(self.num_epochs)):
            #train...
            wandb.log({"loss_" + self.name: loss.item(), "step_" + self.name: i})
```
